src/model.py

```diff
@@ -0,0 +1,92 @@
+
+import argparse
+import logging.config
+import os
+import sys
+
+import sqlalchemy
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, String, MetaData
+
+sys.path.append(os.path.abspath(os.path.join('..')))
+
+from config import SQLALCHEMY_DATABASE_URI, DATABASE_NAME
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel("INFO")
+
+Base = declarative_base()
+
+
+class Userinput(Base):
+
+    """Create a data model for the database to be set up for capturing user input
+
+    """
+
+    __tablename__ = 'Userinput'
+
+    id = Column(Integer, primary_key=True)
```

Comment:
Unless id is explicitly a part of input, a suggestion is to make this autoincrement.